examples/ESP8266_NoteOnOffEverySec/ESP8266_NoteOnOffEverySec_softAP_mDNS.ino

```diff
@@ -0,0 +1,96 @@
+// Example to start ESP8266 in soft access point / hotspot mode
+// and also enable mDNS response on local network. This allows
+// client to discover the AppleMIDI service and connect to it
+// without having to type the IP address and port
+// Tested on iOS 9 (old iPad) and iOS 13 (iPhone 6)
+// On Win10 (rtpMIDI), ESP8266 did not show in directory,
+// but connects fine with default IP(192.168.4.1)/port(5004)
+
+#include <ESP8266WiFi.h>
+#include <ESP8266mDNS.h>
+#include <WiFiClient.h>
+#include <WiFiUdp.h>
+
+#define SerialMon Serial
+#define APPLEMIDI_DEBUG SerialMon
+#include <AppleMIDI.h>
+
+char ssid[] = "ESP8266"; //  your network SSID (name)
```

Comment:
char ssid[] = "ssid"; //  your network SSID (name)

Reply:
Amended previous commit. That was not my SSID/password - tried to use something generic. Anyway - I should have stuck to your defaults :-)